Aspire Budgeting/Views/CardView/BaseCardView.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  BaseCardView.swift
+//  Aspire Budgeting
+//
+
+import SwiftUI
+
+struct BaseCardView<Content: View>: View {
+  var minY: CGFloat = 0
+  var curY: CGFloat = 0
+
+  private let cornerRadius: CGFloat = 24
+  private let height: CGFloat = 163
+
+  private let shadowRadius: CGFloat = 14
+  private let shadowYOffset: CGFloat = 4
+
+  private var baseColor: Color
+  private let content: Content
+
+//  private var gradient: LinearGradient {
+//    Color.fondGradientFrom(startColor: colorInfo.gradientStartColor,
+//                           endColor: colorInfo.gradientEndColor)
+//  }
+
+  private var offsetY: CGFloat {
+    curY < minY ? minY - curY : 0
+  }
+
+  init(minY: CGFloat = 0,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/ab414e6b821895ee43e8c1c80baa6a7893f3ce2e/Aspire Budgeting/Views/CardView/BaseCardView.swift#L30"></span>Prefer to locate parameters with defaults toward the end of the parameter list.<br />
<code>function_default_parameter_at_end</code> <code>BaseCardView.swift:30</code></td>
    </tr>
  </tbody>
</table>